test/test_jit.py

```diff
@@ -96,6 +96,21 @@ def add_kwargs(first, second): return (first+second).realize()
       np.testing.assert_allclose(c.numpy(), a.numpy()+b.numpy(), atol=1e-4, rtol=1e-5)
     assert_jit_cache_len(add_kwargs, 1)
 
+  def test_reorder_kwargs_jit(self):
+    @TinyJit
+    def add_kwargs(first, second): return (first+second).realize()
```

Comment:
Can you make the test something non commutative so we can confirm the order is right?

Reply:
Actually, good catch. `first/second` breaks for example. Let me see if there's a better fix.